packages/gatsby-cli/src/reporter/redux/index.js

```diff
@@ -0,0 +1,73 @@
+const Redux = require(`redux`)
```

Comment:
Aha, that's what I thought. Heh.

Reply:
And done

Reply:
Moved back to `redux` because I changed my mind and think we need to come up with a better name hehe 😄 

Or probably just move the wrapper over to a real middleware later